docs/source/troubleshooting/general.rst

```diff
@@ -8,6 +8,13 @@ If you forgot your credentials to the web UI, try this.
 
       sunshine --creds {new-username} {new-password}
 
+.. note::
+    If you installed Sunshine via flatpak, try this instead.
```

Comment:
Let's use tabs instead.

```rst
If you forgot your credentials to the web UI, try this.
   .. tab:: General

      .. code-block:: bash

         sunshine --creds {new-username} {new-password}

   .. tab:: AppImage (Linux)

      .. code-block:: bash

         ./sunshine.AppImage --creds {new-username} {new-password}

   .. tab:: Flatpak (Linux)

      .. code-block:: bash

         flatpak run --command=sunshine dev.lizardbyte.Sunshine --creds {new-username} {new-password}
```

Note: I also changed the flatpak to the one we distribute in GitHub releases. `dev.lizardbtye.app.Sunshine` is the community version. I have a goal to get to parity with the flathub version, but for now I want our docs referencing what we control in our repo.